sys/freebsd/sys.txt

```diff
@@ -21,6 +21,8 @@ include <signal.h>
 include <sys/wait.h>
 include <sys/time.h>
 
+type signo int32[SIGRTMIN:SIGRTMAX]
```

Comment:
I think it should be `0:SIGRTMAX`. `SIGRT*` is the range of "real-time" signals, but we want all signals.